src/MSBuild.Sdk.SqlProj.Aspire/DataTierApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,65 @@
+using Aspire.Hosting.ApplicationModel;
+using Aspire.Hosting.Lifecycle;
+using Microsoft.Build.Locator;
+using MSBuild.Sdk.SqlProj.Aspire;
+
+namespace Aspire.Hosting;
+
+public static class DataTierApplicationBuilderExtensions
+{
+    /// <summary>
+    /// Adds a data-tier application resource to the application based on a referenced project.
```

Comment:
Maybe: "Adds a data-tier application resource to the application based on a referenced MSBuild.Sdk.SqlProj project." ?